README.md

```diff
@@ -49,7 +49,7 @@ Pipeline execution
 Trigger nightly pipeline manually
 ---
 ```shell
-kubectl create job --from=cronjob/trigger-nightly-pipeline trigger-nightly-pipeline-$(date +%d_%m)-$(whoami)-manual -n ${PIPELINE_NAMESPACE}
+kubectl create job --from=cronjob/trigger-nightly-pipeline trigger-nightly-pipeline-$(date +%d.%m)-$(whoami)-manual -n ${PIPELINE_NAMESPACE}
```

Comment:
`.` doesnt work, what about `-`?

Reply:
what do you mean it doesnt work?

Reply:
My mistake. It works, it just throws a warning. :+1: 
```
Warning: metadata.name: this is used in Pod names and hostnames, which can result in surprising behavior; a DNS label is recommended: [must not contain dots]
```